.cirrus.yml

```diff
@@ -1,158 +1,170 @@
-        cd $ENGINE_PATH/src/flutter/lib/web_ui
-        $ENGINE_PATH/src/out/host_debug_unopt/dart-sdk/bin/pub get
-        export FELT="$ENGINE_PATH/src/out/host_debug_unopt/dart-sdk/bin/dart dev/felt.dart"
-        $FELT test --browser=firefox
-    - name: build_and_test_android_unopt_debug
-      env:
-        USE_ANDROID: "True"
-        ANDROID_HOME: $ENGINE_PATH/src/third_party/android_tools/sdk
-      lint_host_script: |
-        cd $ENGINE_PATH/src/flutter/tools/android_lint
-        $ENGINE_PATH/src/third_party/dart/tools/sdks/dart-sdk/bin/pub get
-        $ENGINE_PATH/src/third_party/dart/tools/sdks/dart-sdk/bin/dart bin/main.dart
-      compile_host_script: |
-        cd $ENGINE_PATH/src
-        ./flutter/tools/gn --android --unoptimized
-        ninja -C out/android_debug_unopt
-        mkdir javadoc_tmp
-        ./flutter/tools/gen_javadoc.py --out-dir javadoc_tmp
-      test_android_script: cd $ENGINE_PATH/src && python ./flutter/testing/run_tests.py --type=java
-    - name: format_and_dart_test
-      format_script: |
-        cd $ENGINE_PATH/src/flutter
-        ./ci/format.sh
-      build_script: |
-        cd $ENGINE_PATH/src/flutter
-        ./ci/build.sh
-    - name: build_fuchsia_artifacts
-      compile_fuchsia_script: |
-        cd $ENGINE_PATH/src
-        ./flutter/tools/fuchsia/build_fuchsia_artifacts.py --engine-version HEAD --runtime-mode debug --no-lto --archs x64
-        cd $ENGINE_PATH/src/flutter
-
-# WINDOWS
-task:
-  gce_instance:
-    image_project: flutter-cirrus
-    image_name: flutter-engine-windows-server-2016
-    zone: us-central1-a
-    platform: windows
-    cpu: 32
-    memory: 32Gb
-    disk: 50
-  env:
-    # Cirrus is somehow not picking up the environment variables set in the VM image.
-    PATH: "c:/depot_tools;c:/MinGit/cmd;$PATH"
-    DEPOT_TOOLS_WIN_TOOLCHAIN: 0
-    GYP_MSVS_OVERRIDE_PATH: "c:/Program Files (x86)/Microsoft Visual Studio/2017/Community"
-    ENGINE_PATH: "c:/flutter/engine"
-  setup_script: |
-    REM robocopy can return 1 for successful copy; suppress its error code.
-    REM move somehow doesn't work as it complains that the file is being used by another process.
-    robocopy %CIRRUS_WORKING_DIR% %ENGINE_PATH%/src/flutter /MIR| (cmd /s /c exit /b 0)
-    cd %ENGINE_PATH%/src
-    gclient sync
-  matrix:
-    - name: build_and_test_windows_unopt_debug
-      compile_host_script: |
-        cd %ENGINE_PATH%/src
-        python flutter/tools/gn --runtime-mode debug --unoptimized
-        ninja -C out/host_debug_unopt
-      test_host_script: |
-        cd %ENGINE_PATH%/src
-        python flutter/testing/run_tests.py --type=engine
-    - name: build_windows_opt_debug
-      compile_host_script: |
-        cd %ENGINE_PATH%/src
-        python flutter/tools/gn --runtime-mode debug
-        ninja -C out/host_debug
+gcp_credentials: ENCRYPTED[987a78af29b91ce8489594c9ab3fec21845bbe5ba68294b8f6def3cf0d380830f06687a89ea69c87344c5ade369700fe]

```

Comment:
Did you edit this file on Windows? I can't exactly tell by the diff shows on Github, but I think you might have accidentally changed the newline style from Linux `\n` to Windows `\r\n`.

Reply:
thanks for noticing this! I changed them back to LF :)